pom.xml

```diff
@@ -19,35 +19,32 @@
     <description>A Mule connector that provides functionality to manage files in an FTP server</description>
 
     <properties>
```

Comment:
@chirag868 I think we have reverted the parent pom version: mule-core-modules-parent. It should be 1.3.2

Reply:
upgraded to 1.3.2